cvmfs/publish/cmd_commit.cc

```diff
@@ -0,0 +1,103 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#include "cvmfs_config.h"
+#include "cmd_commit.h"
+
+#include <errno.h>
+
+#include <ctime>
+#include <string>
+#include <vector>
+
+#include "logging.h"
+#include "publish/cmd_util.h"
+#include "publish/except.h"
+#include "publish/repository.h"
+#include "publish/settings.h"
+#include "util/posix.h"
+#include "util/string.h"
+#include "whitelist.h"
+
+namespace publish {
+
+int CmdCommit::Main(const Options &options) {
+  // Repository name and lease path are submitted as a single argument
```

Comment:
For the commit command, there is only the `fqrn` but not the `fqrn/lease_path` notation; the latter exists for opeining transactions only.